### PR TITLE
Translation of sustainability chapter to Japanese

### PR DESCRIPTION
**Pull request by ksakae1216**
Please review for me.
Part of [#586](https://github.com/HTTPArchive/almanac.httparchive.org/issues/586)

### CODE REVIEW

File: src/content/ja/2022/sustainability.md [hunk 1]
```diff
@@ -0,0 +1,814 @@
...
+  caption="モバイルでのCMSおよびリソースタイプ別の中央値キロバイト",
+  description="",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQvn7rDUZ96mbcJGd-R-gGdofTptGuReAxtTp-jYGUPpXaDO11ef2LjXz_aj-bk7wIA3gvFbEX_El-e/pubchart?oid=1726838193&format=interactive",
+  sheets_gid="1561070567",
+  sql_file="cms_bytes_per_type.sql"
+) }}
+
+上の表およびグラフは、Wixがモバイルイメージの最適化により積極的に取り組んでいることを示しています。Next.jsやNuxt.jsのようなフレームワークを使用しているサイトジェネレーターのセグメントでも同様のパターンが見られます。
+
+この見かけ上小さな洞察は、プラットフォームおよびフレームワークがより持続可能なウェブサイトを提供するために果たす重要な役割を捉えています。適切なデフォルトを適用することにより、プラットフォーム開発者およびフレームワークの著者は、開発者がツールを活用して、_<a hreflang="en" href="https://screenspan.net/blog/green-by-default/">green by default</a>_のサイトを作成できるように支援できます。
```
Comment (tunetheweb) [inline src/content/ja/2022/sustainability.md:780]: Does "green by default" need a translation or is it OK as is?

File: src/content/ja/2022/sustainability.md [hunk 2]
```diff
@@ -0,0 +1,814 @@
...
+  <table>
+    <thead>
+      <tr>
+        <th scope="col">CMS</th>
+        <th scope="col">Device</th>
+        <th scope="col">HTML</th>
+        <th scope="col">JavaScript</th>
+        <th scope="col">CSS</th>
+        <th scope="col">Image</th>
+        <th scope="col">Fonts</th>
```
Comment (tunetheweb) [inline src/content/ja/2022/sustainability.md:661]: Do some of these need translated? For example Device probably should be translated yet? the others are maybe OK as technical terms.

**Review by tunetheweb — COMMENTED** [review]
LGTM with a couple of questions

File: src/content/ja/2022/sustainability.md (hunk 1 above)
Comment (ksakae1216) [reply src/content/ja/2022/sustainability.md]: Thank you for your question. The phrase "green by default" does not need to be translated

File: src/content/ja/2022/sustainability.md (hunk 2 above)
Comment (ksakae1216) [reply src/content/ja/2022/sustainability.md]: That's a challenging question. I've pondered it myself.
However, I believe I have been translating "device" into Japanese, so I will translate it.
https://github.com/HTTPArchive/almanac.httparchive.org/pull/3772/commits/7711605a29947e5dd00924b13c2cf80cd82b5b03

**Review by ksakae1216 — COMMENTED** [review]
@tunetheweb Thanks, I translated.
Please re-review for me.

**Pull request merged**